cinch/bin/wrappers.py

```diff
@@ -3,14 +3,14 @@
 from contextlib import contextmanager
 from plumbum import local
 from plumbum.commands.processes import ProcessExecutionError
-from os import path, chdir, getcwd
```

Comment:
Mainly for convenience.  As discussed we will leave it be and just import all of the 'os' module.